### PR TITLE
Update architecture docs

### PR DESCRIPTION
**Pull request by TimboTambo**
# Update architecture docs

Update architecture docs now that the PARs site has been released.

### Acceptance Criteria

- [ ] Docs reflect the current architecture

### Testing information

_notes that might be helpful for testing_

### Pre-flight Checklist

- [ ] Testing aligns with [testing strategy][testing strategy]
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"

### CODE REVIEW

File: medicines/docs/architecture/README.md [hunk 1]
```diff
@@ -2,34 +2,46 @@
...
 
 Metadata for the medicines (including lists of associated documents) is attached to each file. Azure Search indexes the documents (pdf and Blob Storage are both supported). Azure search also has AI plugins for features such as phrase detection. This has the potential to give us a rich, relevant search experience similar to that provided by commercially available search engines such as Google or Bing).
 
-The API pod contains a lightweight custom HTTP server, written in Rust. It is stateless so it can scale out easily. It aggregates data from all three sources and presents a documented API that conforms to the [Open API 3.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md). It will be read-only to start with.
+## Medicines API
+
+The API pod contains a lightweight custom HTTP server, written in Rust. It is stateless so it can scale out easily. It allows SPC, PIL and PAR to be searched via a self-documented, read-only, [GraphQL-based API](https://medicines.api.mhra.gov.uk/graphiql) that conforms to the [Open API 3.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md).
```
Comment (StuartHarris) [inline medicines/docs/architecture/README.md:19]: I think we should remove the reference to OpenAPI 3.0 which really only applies to REST apis

File: medicines/docs/architecture/README.md [hunk 2]
```diff
@@ -2,34 +2,46 @@
...
 
-The Web pod is the server-side rendering (SSR) part of the `next.js` client-side web application. SSR gives us better SEO and a better user experience (faster initial page load) etc.
+Authentication and authorization is provided by Azure's Active Directory (AD) service. The email addresses of approved medical writers are managed by a group within Azure AD that authorises them to edit PARS.
 
-The `doc-index-updater` pod is an API for the Sentinel Batch export. It is written in Rust and provides an API for Sentinel batch to call, validates and processes documents before storing data and PDFs to Blob Storage and Azure Search.
+Login requests are validated by Azure AD and an ID JWT token returned. Once completed, the PAR form is submitted, along with the JWT token, to a PARs endpoint on the Document Index Updater. An authorization policy in Istio ensures that any requests to this endpoint are accompanied by a valid, Azure-signed JWT token that contains the necessary claim to make changes to PARS (granted by membership of the group, defined within AD).
```
Comment (StuartHarris) [inline medicines/docs/architecture/README.md:29]: Are we actually checking claims? If we are that's great! But I seem to recall there was an issue getting the claim into the token, so now you can only get a token if you're in the group and we accept all tokens from this issuer and application?

Comment (TimboTambo) [reply medicines/docs/architecture/README.md:29]: Ah that's correct - yes will amend wording here.

File: medicines/docs/architecture/README.md (hunk 1 above)
Comment (TimboTambo) [reply medicines/docs/architecture/README.md:19]: Agreed.

**Pull request merged**